### ISSUE TITLE
What a cool idea...

### ISSUE THREAD

**Issue by theopensourcerer** [theopensourcerer/esp8266-programmer#11]
@beetecservices I know this kind of went to sleep but I just saw this and it reminded me. I'll surely need one of these sooner or later.

https://www.youtube.com/watch?v=vXDHyCQ3suw

**Comment by beetecservices** [comment]
Yeah, I can't remember how far we'd got - my routine was thrown out of
whack when I was made redundant and my project work went with it.

It's pretty cool - I wonder if paper-clips could be adapted in the same
way. Obviously not as springy as safety pins but it could work.

**Comment by theopensourcerer** [comment]
Yes - me too. Will revisit this in the new year. I wanted to record that link before I forgot about it! 

Have a great Christmas & New Year Brian.
